Create an overview of all recurring calls (#250)
solid/solidproject.org: pages/standardisation.md

Anyone can contribute to the Solid Specification by making suggestions by submitting [issues](https://github.com/solid/specification/issues) and [pull requests](https://github.com/solid/specification/pulls). Before contributing to the Solid Specification please read the [Code of Conduct](#conduct) and [join the W3C Solid Community Group](https://www.w3.org/community/solid/). 

> It is suggested that you present your proposals at the relevant calls and communication channels, which include the following weekly recurring meetings: 

Reviewer: ```suggestion
You are warmly invited to present your proposals at the relevant calls and communication channels, which include the following weekly recurring meetings: 
```

> * Mondays at 1600 CEST Authentication Panel on [this line](https://inrupt.my.webex.com/inrupt.my/j.php?MTID=m131f1588eb27afc298276f1cbf077a79)

Reviewer: ```suggestion
* Mondays at [1400 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20200907T14) Authentication Panel on [this line](https://inrupt.my.webex.com/inrupt.my/j.php?MTID=m131f1588eb27afc298276f1cbf077a79)
```
